src/db/commit.rs

```diff
@@ -183,7 +183,11 @@ impl Commit {
         let commit = commit_fb::Commit::create(&mut builder, commit_args);
         builder.finish(commit, None);
 
-        let chunk = dag::Chunk::new(builder.collapse(), &[value_hash]);
+        let refs = std::iter::once(value_hash)
```

Comment:
Fancy. Another way to say it would just be:

```
let mut refs = vec![value_hash];
if let Some(basis_hash) = basis_hash {
  refs.push_back(basis_hash);
}
```

Reply:
I had this before and switch to the `once` to keep the two places more similar